Src/Translations/Resources/hi.ts

```diff
@@ -81,4 +81,15 @@ export default {
                 `,
     },
   },
+  currentProvider: {
```

Comment:
can you please convert this text as per language.

Reply:
done, convert into english to hindi